README.md

```diff
@@ -2,8 +2,8 @@
 
 ## Deploying
 
-Needs a Postgres database. URL is passed in as `DATABASE_URL`. For _reasons_,
-don't use `postgres://postgres@localhost/postgres`.
+Needs a Postgres database. URL is passed in in the `control.yaml` file. For
```

Comment:
This should probably be either `control.toml` or `compose.yaml`

Reply:
Yeah, sleepy misspelling on my part, will fix soon

Reply:
Fixed in 7e12fd7

Reply:
Fixed in 762600c